src/WorkMonth.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+
+import MonthView from './Month'
+import cn from 'classnames'
+import * as dates from './utils/dates'
+import chunk from 'lodash/chunk'
+import Header from './Header'
+
+const isWorkDay = date => {
+  return date.getDay() !== 6 && date.getDay() !== 0
+}
+
+const visibleWorkDays = function(date, localizer) {
+  let current = dates.firstVisibleDay(date, localizer),
+    last = dates.lastVisibleDay(date, localizer),
+    days = []
+
+  while (dates.lte(current, last, 'day')) {
+    days.push(current)
+    current = dates.add(current, 1, 'day')
+  }
+
+  days = days.filter(date => isWorkDay(date))
+
+  return days
+}
+
+const workDaysRange = function(start, end, unit = 'day') {
+  let current = start,
+    days = []
+
+  while (dates.lte(current, end, unit)) {
+    days.push(current)
+    current = dates.add(current, 1, unit)
+  }
+
+  days = days.filter(date => isWorkDay(date))
+
+  return days
+}
+
+class WorkMonthView extends MonthView {
```

Comment:
can we change the Monthview api such that you can compose them normally instead of inheritance?

Reply:
Hi Jquense,

Good idea, much cleaner indeed. Just didn't want to modify your cadebase.
I've now created a pull request that moves the logic into the month view component.

When the component is initialized with the 'workdaysOnly' prop, it will hide the weekends

Reply:
Hi Jquense,

So you see some time to review the new pull request? 
Thanks.

Kind regards,
Jonas